### ISSUE TITLE
regarding running training script

### ISSUE THREAD

**Issue by huaiyanggongzi** [sagieppel/Reconstruct-image-from-sparsely-sampled-pixels-using-fully-convolutional-neural-network-FCN-#1]
Hi, thanks for sharing the code.

I am running the train script, by using `python Train.py`. Looks like it is running, are there any ways to monitor this running process, like printing out how many training iterations having been passed? I am not sure how long it will take for finishing the training.

This the current output,

`python Train.py
Finished Running
setting up vgg initialized conv layers ...
Reading images list
Number of  Train images=20210
2017-07-04 15:16:14.178105: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
2017-07-04 15:16:14.178130: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
2017-07-04 15:16:14.178138: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.

`

**Comment by sagieppel** [comment]
The 3 final  messages are related to Tensorflow installation  and shouldn't effect you if you use GPU (if you dont use GPU training will take you forever).
The training loss should be printed on screen once every ten training operation, which should accure once every few seconds, if it doesn't appear something strange happening.
